contracts/schemes/Auction4Reputation.sol

```diff
@@ -38,7 +39,7 @@ contract Auction4Reputation {
     address public wallet;
 
     /**
-     * @dev constructor
+     * @dev setParameters
```

Comment:
The name `setParameters` is an abuse of notation, as we use it in Arc for a different concept.
I think it is better to change it to `initialize` or something similar.

Reply:
done at f98d948